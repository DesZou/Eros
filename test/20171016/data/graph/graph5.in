50 120 48
20 9 342
25 27 712
13 1 23
11 48 654
37 34 707
4 19 944
6 11 943
8 3 2
29 48 119
2 10 426
16 14 604
45 36 348
30 35 188
1 2 826
19 13 909
34 9 955
2 13 523
31 45 10
36 48 305
11 45 405
22 13 983
21 29 913
50 33 345
49 32 256
49 46 704
36 30 343
36 38 309
19 35 742
42 7 952
19 5 922
13 3 656
17 1 46
27 40 952
32 31 945
4 24 939
33 16 546
9 47 703
44 24 985
17 31 822
12 14 781
36 11 940
49 1 28
25 37 431
35 17 571
34 40 935
31 40 492
25 44 4
1 9 805
10 1 718
23 18 161
40 25 219
34 17 45
47 33 956
30 11 233
40 49 41
22 42 366
35 22 709
44 20 849
14 50 435
10 17 593
47 31 335
21 9 770
42 33 456
45 43 742
14 5 423
8 48 680
1 47 665
11 43 512
10 5 497
16 24 538
45 21 34
5 7 270
15 16 65
41 31 958
43 31 419
27 23 665
14 22 306
18 29 415
1 22 820
16 20 913
25 35 135
34 14 162
5 39 212
36 34 993
6 15 416
31 35 346
26 2 718
5 43 696
13 25 594
4 23 989
18 38 76
40 21 174
47 26 384
41 49 402
1 24 736
24 25 41
2 6 136
41 34 673
6 47 963
26 29 308
6 29 339
15 43 899
12 30 802
43 39 319
17 4 398
46 7 621
24 3 339
29 46 575
32 16 733
10 37 431
29 4 107
33 44 136
50 45 198
44 28 888
50 37 607
18 4 983
15 7 163
21 24 796
26 49 790
46 40 166
50
7 12
6 14
30 50
46 18
44 5
6 9
19 42
37 2
22 1
19 35
11 42
39 37
13 10
13 35
1 50
47 29
12 41
34 22
34 27
12 11
10 23
15 30
23 41
32 30
25 41
12 23
2 23
11 26
31 48
19 1
28 15
13 35
33 5
4 39
10 23
49 28
11 48
25 15
14 1
48 47
40 30
46 21
47 49
25 43
2 36
48 42
39 44
43 2
17 7
24 15
