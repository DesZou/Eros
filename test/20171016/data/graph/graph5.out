2199
1655
1116
1527
1128
2215
1976
1433
1287
1523
1320
1986
1341
913
941
1714
1955
1370
1259
1849
2190
1242
1945
1392
1220
2565
2097
1574
1498
1668
1811
913
1484
2182
2190
1328
1666
927
1318
1438
1177
848
1381
1244
1660
1706
1559
1652
1301
861
